.github/workflows/dev-container.yml

```
@@ -26,13 +28,51 @@ jobs:
+        uses: devcontainers/ci@v0.3
+        with:
+          cacheFrom: ghcr.io/partcad/partcad-devcontainer:${{ github.event.pull_request.number }}
+          push: never
+          runCmd: poetry --no-ansi --no-interaction install
+  pytest:
+    name: Run pytest
+    runs-on: ubuntu-latest
+    needs: install
+    steps:
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)